src/components/Editor/Preview/Popup.js

```diff
@@ -73,10 +78,17 @@ function inPreview(event) {
   return inPreviewSelection;
 }
 
-export class Popup extends Component<Props> {
+export class Popup extends Component<Props, State> {
   marker: any;
   pos: any;
 
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      top: 0
+    };
+  }
```

Comment:
I wonder if this can be:

```js
export class Popup extends Component<Props, State> {
  // stuff
   state: State = { top: 0 }
   // stuff
```


```